git/git.go

```diff
@@ -47,5 +48,7 @@ func GetVersions(ctx context.Context, pckg *packages.Package, packageGitcache st
 		})
 	}
 
-	return gitVersions
+	// TODO, determine latest git tag
+	// https://github.com/cdnjs/tools/issues/39
+	return gitVersions, ""
```

Comment:
Will this set an empty value to the version field? we need to keep the exiting behavior for git

Reply:
Yes I was a bit confused here.

> For Git:
That version is the latest version sort by date, which includes any non stable versions. We should parse semver and get the most recent that parsed and is stable.

Is the `latest` git version just the most recent one by date that is also semver-parsable? I'm a bit caught up on "and is stable".